### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lasomethingsomething**
Reflects changes Dan and I discussed last Friday. LMK if there are any issues or questions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,17 @@
-# Beard Template Engine
+# Beard: A Blazing-Fast Template Engine
 
 [![Build Status](https://travis-ci.org/zalando/beard.svg)](https://travis-ci.org/zalando/beard)
 
-Blazing fast, [open source](https://github.com/zalando/beard), logic-less template engine written in Scala, used to parse templates using a simple syntax:
+Beard is an open-source, logic-less templating engine, written in Scala and inspired by [Mustache](https://mustache.github.io/). You can use it out-of-the-box; it has no dependencies (see Requirements list below).
```
Comment (danpersa) [inline README.md]: We shouldn't specify that it doesn't have dependencies, as it has some. For example the streaming library.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,17 @@
...
 
 [![Build Status](https://travis-ci.org/zalando/beard.svg)](https://travis-ci.org/zalando/beard)
 
-Blazing fast, [open source](https://github.com/zalando/beard), logic-less template engine written in Scala, used to parse templates using a simple syntax:
+Beard is an open-source, logic-less templating engine, written in Scala and inspired by [Mustache](https://mustache.github.io/). You can use it out-of-the-box; it has no dependencies (see Requirements list below). 
+
+What makes Beard powerful:
+
+  - Streaming. As soon as you need to render something, you can stream it to the browser. This provides high user-perceived performance.
+  - Speed. We've benchmarked it against other template engines for the JVM, and Beard performed twice as fast in terms of rendering time. (We invite you to run your own benchmarks to see if you get the same results.) It also uses ANTRL to make template compilation fast.
```
Comment (danpersa) [inline README.md]: I wouldn't say twice as fast, as this is hard to prove in every case... I'd say `faster than other template engines`

File: README.md [hunk 3]
```diff
@@ -15,46 +24,56 @@ Blazing fast, [open source](https://github.com/zalando/beard), logic-less templa
...
-  - streaming: as soon as we have something to be rendered, we are able to stream it to the browser. This gives high user perceived performance.
-  - fast: have benchmarked it against other jvm template engines, and we strive to keep it on top. Check out our [repository](https://github.com/zalando/beard) to run the benchmarks.
-  - beautiful: only using brackets for delimiters keeps it beautiful
+## Installing
+If you're using SBT, add this line to your build.sbt file:
 
-Uses ANTRL to compile the templates which makes the compilation really fast.
+    libraryDependencies += "de.zalando" %% "beard" % "0.0.6"
+
+Binaries are available from [bintray](https://bintray.com/zalando-spearheads/java/beard/0.0.2/view). Or simply
```
Comment (danpersa) [inline README.md]: These two lines should be replaced with:

```
Binaries are available from [bintray](https://bintray.com/zalando/maven/beard/). Or simply add `resolvers += Resolver.bintrayRepo("zalando", "maven")` to your `build.sbt`.
```

File: README.md [hunk 4]
```diff
@@ -15,46 +24,56 @@ Blazing fast, [open source](https://github.com/zalando/beard), logic-less templa
...
+add `resolvers += Resolver.bintrayRepo("zalando-spearheads", "java")` to your `build.sbt`.
 
-More details in our [documentation](https://danpersa.gitbooks.io/beard/content/)
+If you're using Maven, run this:
```
Comment (danpersa) [inline README.md:40]:
```
 If you're using maven add this to your `pom.xml` file
```

File: README.md [hunk 5]
```diff
@@ -15,46 +24,56 @@ Blazing fast, [open source](https://github.com/zalando/beard), logic-less templa
...
 
-# Gitbook
+    <dependency>
+    	<groupId>de.zalando</groupId>
+    	<artifactId>beard</artifactId>
+    	<version>0.0.6</version>
+    </dependency>
 
-Contribute to our gitbook [here](https://github.com/danpersa/beard-book).
+Or simply add:
```
Comment (danpersa) [inline README.md]: we don't need these lines with the resolver

**Comment by lasomethingsomething** [comment]
merged

**Pull request merged**